infra/core/tenant/role-definitions/flexible-server.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
```

Comment:
should be bicepparams

Reply:
There was an issue bicep param, the array was not working. So I have created json file for now.